silk/device/netns_base.py

```diff
@@ -78,7 +79,10 @@ def create_netns(self):
 
         self.netns = os.path.basename(self.device_path)
         command = "sudo ip netns add %s" % self.netns
+        print('+'*100)
+        print(command)
```

Comment:
please remove. They are included in the log.

Reply:
Done